EventPlanning.service/Program.cs

```diff
@@ -18,7 +18,7 @@ public static void Main(string[] args)
                 try
                 {
                     var context = services.GetRequiredService<DAL.EventContext>();
-                    context.Database.EnsureDeleted();
+                    //context.Database.EnsureDeleted();
```

Comment:
Let's not leave comments. Writing it several times will make you remember the stuff.
There is a concept here that is interesting, quick read:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if